dart/lib/src/sentry_envelope.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:convert';
+
+import 'sentry_envelope_header.dart';
+import 'sentry_envelope_item.dart';
+import 'protocol/sentry_event.dart';
+import 'protocol/sdk_version.dart';
+
+/// Class representation of `Envelope` file.
+class SentryEnvelope {
+  SentryEnvelope(this.header, this.items);
+
+  /// Header descriping envelope content.
+  final SentryEnvelopeHeader header;
+
+  /// All items contained in the envelope.
+  final List<SentryEnvelopeItem> items;
+
+  /// Create an `SentryEnvelope` with containing one `SentryEnvelopeItem` which holds the `SentyEvent` data.
+  factory SentryEnvelope.fromEvent(SentryEvent event, SdkVersion sdkVersion) {
+    return SentryEnvelope(SentryEnvelopeHeader(event.eventId, sdkVersion),
+        [SentryEnvelopeItem.fromEvent(event)]);
+  }
+
+  /// Stream binary data representation of `Envelope` file encoded in utf8.
+  Stream<List<int>> envelopeStream() async* {
```

Comment:
`async*` what is the meaning of `*` here?

Reply:
It returns an async stream. This enables you to use `yield` and `await` in the function block.

Reply:
TIL